src/test/java/org/folio/marc4ld/mapper/test/MonographTestUtil.java

```diff
@@ -628,6 +620,26 @@ public static Resource createSampleWork() {
     ).setLabel("Work: label");
   }
 
+  public static Resource status(String value) {
```

Comment:
minor - can we make this method to accept a boolean value `isCancelled`? This way, we can avoid passing string "canceled or invalid" and matching against it in line 628.

Reply:
I suppose that we pass it to use it for the label. So I've left it as is for now.